packages/docusaurus-preset-classic/package.json

```diff
@@ -29,7 +29,7 @@
     "@docusaurus/theme-search-algolia": "2.0.0-beta.10"
   },
   "peerDependencies": {
-    "@docusaurus/core": "2.0.0-beta.9",
+    "@docusaurus/core": "2.0.0-beta.10",
```

Comment:
maybe we should make it an range? ^2.0.0

Reply:
I don't think ranges work for beta tags. But I could be wrong

Reply:
Caret ranges do work: https://docs.npmjs.com/cli/v6/using-npm/semver#prerelease-tags

I will probably be going for a range, because if lerna doesn't want to update peer dependencies (https://github.com/facebook/docusaurus/pull/6047#issuecomment-989855782) we'd have to make it future-proof